sdk/cognitivelanguage/ai-language-conversations/samples-dev/sample_analyze_orchestration_app_conv_response.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to analyze user query using an orchestration project.
+ * In this sample, orchestration project's top intent will map to a conversation project.
+ *
+ * @summary Orchestration project with conversational response
+ * @azsdk-weight 50
+ */
+
+import { ConversationAnalysisClient, ConversationalTask } from "@azure/ai-language-conversations"
+import { AzureKeyCredential } from "@azure/core-auth";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+//Get secrets
+//You will have to set these environment variables for the sample to work
+const clu_endpoint = process.env.AZURE_CONVERSATIONS_ENDPOINT || "https://dummyendpoint.cognitiveservices.azure.com";
+const clu_key = process.env.AZURE_CONVERSATIONS_KEY || "<api-key>";
+const project_name = process.env.AZURE_CONVERSATIONS_WORKFLOW_PROJECT_NAME || "<project-name>";
+const deployment_name = process.env.AZURE_CONVERSATIONS_WORKFLOW_DEPLOYMENT_NAME || "<deployment-name>";
```

Comment:
Could all variables be renamed tp use came case? e.g. `deployment_name` to `deploymentName`.